app/models/qa/linked_data/config/context_property_map.rb

```diff
@@ -70,12 +98,43 @@ def extract_label
             default
           end
 
-          def ldpath_program
-            return @program if @program.present?
+          def basic_program
+            @basic_program ||= ldpath_program(ldpath)
+          end
+
+          def expansion_label_program
+            @expansion_label_program ||= ldpath_program(expansion_label_ldpath)
+          end
+
+          def expansion_id_program
+            @expansion_id_program ||= ldpath_program(expansion_id_ldpath)
+          end
+
+          def ldpath_program(ldpath)
             program_code = ""
             prefixes.each { |key, url| program_code << "@prefix #{key} : <#{url}> \;\n" }
             program_code << "property = #{ldpath} \;"
-            @program = Ldpath::Program.parse program_code
+            Ldpath::Program.parse program_code
+          rescue => e
+            Rails.logger.warn("WARNING: #{PARSE_LOGGER_ERROR} (ldpath='#{ldpath}')\n    cause: #{e.message}")
+            nil
+          end
+
+          def ldpath_evaluate(program, graph, subject_uri)
+            return PARSE_ERROR_VALUE if program.blank?
```

Comment:
@elrayle Hate to cause issues, but did you mean nil?  Is the empty string a valid value?

Reply:
With 2nd commit, now returns an empty array when an error occurs.